arch/arm/soc/st_stm32/stm32f7/Kconfig.defconfig.series

```diff
@@ -0,0 +1,42 @@
+# Kconfig.defconfig.series - ST Microelectronics STM32F7 MCU line
+#
+# Copyright (c) 2018 Yurii Hamann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_SERIES_STM32F7X
+
+source "arch/arm/soc/st_stm32/stm32f7/Kconfig.defconfig.stm32f7*"
+
+config SOC_SERIES
+	default stm32f7
+
+if USB
```

Comment:
Remove USB as it is not yet tested/implemented on this series (limited to F4 for now)